digest/glossary.js

```diff
@@ -0,0 +1,27 @@
+const utils = require('./utils')
+
+module.exports = () =>
+  utils.readdir('/glossary')
+    .then(convertGlossaryFileNamesToTerms)
+    .then(loadTermValues)
```

Comment:
checkout https://github.com/learnersguild/curriculum/blob/master/digest/modules.js#L14 and make sure we're only getting directories